src/Microsoft.Fx.Portability/BreakingChange.cs

```diff
@@ -87,7 +87,7 @@ public int CompareTo(BreakingChange other)
 
         public override int GetHashCode()
         {
-            return Id.GetHashCode();
+            return Id?.GetHashCode() ?? 0;
```

Comment:
Never mind. No need to modify this. :) The [updated documentation](https://msdn.microsoft.com/en-us/library/system.object.gethashcode(v=vs.110).aspx) is more clear. If two objects are equal then GetHashCode should be the same for both objects.

It would be better to modify [BreakingChanges.Equals()](https://github.com/Microsoft/dotnet-apiport/blob/master/src/Microsoft.Fx.Portability/BreakingChange.cs) function.. since it would be possible to return true on two separate BreakingChanges that did not have their ID set.